ansible/configs/ansible-tower-advanced/env_vars.yml

```diff
@@ -17,8 +17,8 @@
 ### Common Host settings
 
 repo_method: file # Other Options are: file, satellite and rhn
-tower_admin_password: 'c13aa1f099ee8970cd3fe0e1348f95fe'
-tower_version: "3.6.0-1"
+tower_admin_password: 'changeme'
```

Comment:
What about commenting that, and generate the password and print it as user.info  if it is not defined?

Reply:
If you tell me how to do it, I can try, but honestly I was expecting for a quick fix for now.

Reply:
an example https://github.com/redhat-cop/agnosticd/blob/development/ansible/roles/bastion-student-user/tasks/main.yml#L13-L17